src/App.js

```diff
@@ -24,45 +24,89 @@ function App() {
 		user ? setAuthStatus(true) : setAuthStatus(false);
 	});
 
-	return (
-		<Router>
-			<>
-				<MobileLogo alt="mobile logo" src={mobileLogo} />
-			</>
-			<Switch>
-				<Route path="/" exact>
-					{authStatus ? <Redirect to="/home" /> : <LandingPage />}
-				</Route>
-
-				<Route path="/home" exact>
-					{authStatus ? <UserHome /> : <Redirect to="/login" />}
-				</Route>
-
-				<Route path="/add-medication">
-					<AddMedication />
-				</Route>
-
-				<Route path="/signup" exact>
-					{authStatus ? <Redirect to="/" /> : <SignUp />}
-				</Route>
-
-				<Route path="/login">
-					{authStatus ? <Redirect to="/" /> : <Login />}
-				</Route>
-
-				<Route path="/medication-added">
-					<MedicationAdded />
-				</Route>
-				<Route path="/search">
-					<SearchNHS />
-				</Route>
-
-				<Route>
-					<PageNotFound />
-				</Route>
-			</Switch>
-		</Router>
-	);
+	if (authStatus) {
+		return (
+			<Router>
```

Comment:
i don't think the Router component will work here, you may need to put it in the parent (i.e. in index.js). But if it does work that's cool